StaggeredCoupledProcesses/TH/ThermalDensityDrivenFlow2D/thermal_gravity_driven_flow2d.prj

```diff
@@ -0,0 +1,304 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<OpenGeoSysProject>
+    <mesh>mesh2D.vtu</mesh>
+    <geometry>gravity_driven.gml</geometry>
+
+    <processes>
+        <process>
+            <name>HeatConduction</name>
+            <type>HEAT_CONDUCTION</type>
```

Comment:
It is defined in the source code for process type. See: https://github.com/ufz/ogs/blob/master/Applications/ApplicationsLib/ProjectData.cpp#L303 (from line 303).